roles/docker/tasks/main.yml

```diff
@@ -24,9 +24,8 @@
 - name: Install docker-compose.
   pip:
     name:
-      - docker
-      - docker-compose
-    state: latest
+      - docker==3.7.0
+      - docker-compose==1.23.2
```

Comment:
What about a semantic range? `docker>=3.7,<3.8` and `docker-compose>=1.23,<1.24`

Reply:
We could do that, but this isn't really semver you are suggesting. But I'm not sure if the packages follow semver anyways.

Reply:
Yeah, it's semver light :slightly_smiling_face: We could of course do ">=3,<4" and ">=1,<2" too, but for that I'd first check whether the projects really do follow semver.

Reply:
I fixed that on Jan 27 :wink: